esp/esp/program/modules/forms/splashinfo.py

```diff
@@ -33,32 +33,25 @@
 """
 
 from django import forms
-from esp.middleware import ESPError
-from esp.tagdict.models import Tag
 import simplejson as json
 
-"""
-The SplashInfoForm is customizable for different lunch options.
-The choices should be provided in a Tag, splashinfo_choices, which is in
-JSON format.  Example:
-    key = 'splashinfo_choices'
-    value = '{
-        "lunchsat": [["no", "No thanks; I will bring my own lunch"],
-                  ["chicken", "Yes, Chicken"],
-                  ["steak", "Yes, Steak"],
-                  ["spicy_thai", "Yes, Spicy Thai"],
-                  ["veggie", "Yes, Vegetarian"]],
-        "lunchsun": [["no", "No thanks; I will bring my own lunch"],
-                  ["cheese", "Yes, Cheese Pizza"],
-                  ["pepperoni", "Yes, Pepperoni Pizza"]],
-    }'
-
-The interface (including directions, whether to exclude or rename days,
-and costs) should be controlled by overriding the template:
-    program/modules/splashinfomodule/splashinfo.html
-"""
+from esp.accounting.models import LineItemType
+from esp.accounting.controllers import IndividualAccountingController
+
+
+LUNCH_SATURDAY = 'lunchsat'
+LUNCH_SUNDAY = 'lunchsun'
+DISCOUNT_SIBLING = 'siblingdiscount'
+SIBLING_NAME = 'siblingname'
```

Comment:
Why are these variables if the field names are still hard coded into the form?  I would suggest that either the field names themselves become dynamic or the code goes back to using ```'lunchsat'```, ```'lunchsun'```, etc.